modules/install/k3s_node_role.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
It seems that this file is quite/almost similar to `rke2_node_role.sh`. Can we explore if that script can be repurposed for k3s as well and have a common `node_roles.sh` file?

Reply:
fixed